src/sanitizer.js

```diff
@@ -251,7 +267,11 @@ export function sanitizeHtml(html) {
           continue;
         }
         emit(' ');
-        emit(attrName);
+        if (hasOwn(bindAttribsIndices, String(i))) {
```

Comment:
That won't quite work since emitting the saved names directly ignores modifications to the attributes (potentially) made while executing this function, such as in the block that checks for the `a` tag.